### PR TITLE
Deprecate https://unee-t.com/contact-support/

### PR DESCRIPTION
**Pull request by kaihendry**
Instead of a Web form, a mailto: link is used:

* simpler
* works offline
* can be a more familiar interface
* gets the conversation going

Perhaps the body could have more info like error logs, but I am not sure how to whip them out.

### CODE REVIEW

File: imports/ui/side-menu/side-menu.jsx [hunk 1]
```diff
@@ -66,21 +66,23 @@ class SideMenu extends Component {
         })}
         <Divider />
         {this.linkDrawerItem({
-          href: 'https://unee-t.com/contact-support/',
+          href: 'mailto:support@unee-t.com?subject=' + window.location.href + '&body=' + encodeURIComponent(`user: ${user.emails[0].address}
```
Comment (nbiton) [inline imports/ui/side-menu/side-menu.jsx]: Having the email body inlined inside an `encodeURIComponent` call, inlined in a query string parm, inlined in an object sent to a method, inlined inside a JSX snippet... is a bit confusing to read :sweat_smile: 

Can you extract the email body to a separate const that you define earlier or something and use it instead?

File: imports/ui/side-menu/side-menu.jsx [hunk 2]
```diff
@@ -66,23 +82,11 @@ class SideMenu extends Component {
         })}
         <Divider />
         {this.linkDrawerItem({
-          href: 'https://unee-t.com/contact-support/',
+          href: 'mailto:support@unee-t.com?subject=' + window.location.href + '&body=' + encodeURIComponent(supportEmailBody),
           iconName: 'live_help',
           text: 'Support',
           isExternal: true
         })}
-        {this.linkDrawerItem({
```
Comment (nbiton) [inline imports/ui/side-menu/side-menu.jsx]: Why did you remove these 2 links? I thought we wanted them in

Comment (kaihendry) [reply imports/ui/side-menu/side-menu.jsx]: Forum is gone. FAQ has limited utility too. I can add them back @franck-boullier

**Comment by franck-boullier** [comment]
> Forum is gone. 

no pb to remove this

> FAQ has limited utility too. 

I 'm not sure about that...
The pb at this point is that we have no solid data to know if it's used or not.
I personally think it's pretty useful TBH...

I would ask 2 questions:
- Is there any major reason why we should remove it? 
- Is it breaking something or creating an issue that I'm not aware of?

If the answer to these 2 questions is 'no' then I believe we should keep a link to the FAQ/Documentation.
Maybe we can rename it but I believe we have much bigger fishes to fry at this point...

**Comment by kaihendry** [comment]
Better question to ask is whether there is a reason to keep it? I think the value is rather dubious. Can you prove it's actually used? If it's used, surely there is some UI/UX issue?

It's breaking signal by introducing noise is all.

Your call.

**Comment by franck-boullier** [comment]
> Your call.

@kaihendry let's keep it then
